editor.planx.uk/src/@planx/components/shared/BaseOptionsEditor.tsx

```diff
@@ -76,8 +76,9 @@ export const BaseOptionsEditor: React.FC<BaseOptionsEditorProps> = (props) => (
     )}
     {props.showValueField && (
       <DataFieldAutocomplete
+        key={`${props.value.id}-data-field-autocomplete`}
```

Comment:
This clears up another error where if there was an option(s) with a null data `val`, then the autocomplete inputs could unexpectedly shift or nullify when being dragged into a different order - see before: 
![chrome-capture-2025-0-17](https://github.com/user-attachments/assets/d37682fa-2058-4d03-b6ec-50c6abf8481c)